.github/workflows/publish.yml

```diff
@@ -35,16 +36,14 @@ jobs:
         name: Parse tag
         id: parse_tag
         run: |
-          BRANCH_NAME=${GITHUB_REF#refs/heads/}
-          PACKAGE_VERSION=$( sed -n 's/.*"version": "\(.*\)",/\1/p' package.json )
-          echo "BRANCH_NAME: ${BRANCH_NAME}"
-          echo "PACKAGE_VERSION: ${PACKAGE_VERSION}"
-          if [[ ${BRANCH_NAME} == "main" ]]; then
-            SEED_TAG=${PACKAGE_VERSION}
-          elif [[ ${BRANCH_NAME} == "develop" ]]; then
+          echo "GITHUB_EVENT_NAME: ${GITHUB_EVENT_NAME}"
+          echo "GITHUB_REF: ${GITHUB_REF}"
+          if [[ ${GITHUB_EVENT_NAME} == "push" ]]; then
             SEED_TAG=develop
```

Comment:
pushes only happen for develop now so this is fine

Reply:
Amazing 🤩